cmake/dependencies.cmake

```diff
@@ -6,6 +6,7 @@ include(FetchContent)
 
 find_package(BLAS)
 find_package(LAPACK)
+find_package(LAPACKE)
```

Comment:
@mattldawson @sjsprecious  @K20shores LAPACKE is also used in tests. Since LAPACKE does its own tests when included in CMAKE, is it a good idea to remove the unit tests I wrote for LAPACKE?

Reply:
I think it's fine to include your tests - the `find_pacakge()` doesn't need to be in the `if (TUVX_ENABLE_BENCHMARK)` block. I just thought if it was only used for the benchmarking, it could be in that block, but it's ok where it as, too (in my opinion).